app/js/components/breadCrumbs/breadCrumb.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Breadcrumb, BreadcrumbItem } from 'reactstrap';
+import FontAwesome from 'react-fontawesome';
+import breadCrumb from './breadCrumb.css';
+import { Link, browserHistory, withRouter } from 'react-router';
+
+class BreadCrumbComponent extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { location, router } = this.props
+    return (
+      <div>
+        {location.pathname === '/'
+          ?
+          <Breadcrumb>
+            <BreadcrumbItem active> <a id="home" href="https://modules-refapp.openmrs.org/openmrs/index.htm"><FontAwesome className='fa fa-home' name='home' /></a>< FontAwesome name='chevron-right' className='fa fa-chevron-right' />Patient Search</BreadcrumbItem>
```

Comment:
This block of code does what it is intended to but it can be refactored to avoid redundant/ repetitive code. We could manipulate state to only certain attributes of the <BreadcrumbItem /> component that need change. Lines 18-43

Reply:
@paulupendo am not sure i understand what you mean. I am not setting anything to state. I am using the withRouter to detect location directly from the props. and you cannot set tags to state right?

Reply:
The block of code could be refactored as it is so repetitive.